libr/bin/mangling/swift-sd.c

```diff
@@ -171,33 +177,44 @@ static char *swift_demangle_lib(const char *s) {
 	return NULL;
 }
 
+static const char *str_seek (const char *s, int n) {
+	int i;
+	for (i = 0; i < n && *s != 0; i++) {
+		s++;
+	}
+	return s;
+}
+
+static const char *str_removeprefix (const char *s, const char *prefix) {
```

Comment:
@trufae this also seems useful to have in `r_str.h`, wdyt?

Reply:
yes, feel free to tag this with R2_580, because we cant add new apis until 5.8.. well we can, but must be added as `static inline` in the ` .h` files

Reply:
not for now